Web/BootStrap-Assignment1/GridSystem1.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE>
+<html>
+	<head>
+		<title>Grid System</title>
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+		<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
+		<link rel="stylesheet" href="css/bootstrap.min.css" type="text/css">
+		<link rel="stylesheet" href="css/style.css" type="text/css">
+		<script type="text/javascript" src="js/bootstrap.min.js"></script>
+	</head>
+	<body>
+		<h2>Grid on Large Screen</h2>
+		<div class="container">
+			<div class="row">
+				<div class="col-lg-12 column">Row1</div>
+			</div>
+			<div class="row">
+				<div class="col-lg-2 column">Row2</div>
```

Comment:
1. Class name should be related to style 
Example: if you wan to give border to column then use class name like col-borer or border-col